new-backoffice/extending/modals/custom-modals.md

```diff
@@ -61,11 +71,66 @@ class MyDialog extends UmbElementMixin(LitElement) {
     render() {
         return html`
             <div>
-                <h1>My Modal</h1>
+                <h1>${this.modalContext?.data.headline ?? "Default headline"}</h1>
                 <button @click=${this._handleCancel}>Cancel</button>
                 <button @click=${this._handleSubmit}>Submit</button>
             </div>
         `;
     }
 }
 ```
+{% endcode %}
+
+## Register in the extension registry
+
+The modal element needs to be registered in the extension registry. This is done by defining the modal in the manifest file. The `js` property should point to the file that contains the modal element.
+
+```json
+{
+    "type": "modal",
+    "alias": "My.Modal",
+    "name": "My Modal",
+    "js": "../path/to/my-modal.element.js"
+}
+```
+
+## Open the modal
+
+To open the modal, you need to consume the `UmbModalManagerContext` and then use the modal manager context to open a modal. This example shows how to consume the Modal Manager Context:
+
+{% code title="my-element.ts" %}
+```ts
+import { MY_MODAL_TOKEN } from './my-modal.token';
+import { UMB_MODAL_MANAGER_CONTEXT } from '@umbraco-cms/backoffice/modal';
+import { UmbElementMixin } from '@umbraco-cms/backoffice/element-api';
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+@customElement('my-element')
+class MyElement extends UmbElementMixin(LitElement) {
+    #modalManagerContext?: typeof UMB_MODAL_MANAGER_CONTEXT.TYPE;
+
+    constructor() {
+        super();
+        this.consumeContext(UMB_MODAL_MANAGER_CONTEXT, (instance) => {
+            this.#modalManagerContext = instance;
+            // modalManagerContext is now ready to be used.
+        });
+    }
+
+    render() {
+        return html`
+            <uui-button look="primary" label="Open modal" @click=${this._openModal}></uui-button>
+        `;
+    }
+
+    private _openModal() {
+        this.#modalManagerContext?.openModal(MY_MODAL_TOKEN, {
+            data: {
+                headline: "My modal headline",
+            },
+        });
```

Comment:
In this example it complains that the property `openModal` does not exist on type UmbModalManagerContext, however when I change it to `open` like this:

```
private _openModal() {
        this.#modalManagerContext?.open(MY_MODAL_TOKEN, {
            data: {
                headline: "My modal headline",
            },
        });
```

It stops complaining,  howering over it says that it opens a modal or sidebar modal. 

So would it be correct that it needs to be changed to `open`? :)